web-local/.jest/setup-e2e-rill.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+skipUIBuild=false;
+
+while getopts "s" arg; do
+  case $arg in
+    s) skipUIBuild=true;;
+  esac
+done
+
+if ! $skipUIBuild ; then
+    ECHO "Building UI"
+    make cli.prepare -C ../..
```

Comment:
This ends up building the UI twice in CI. We can remove the build from the job config.

Reply:
For CI, you can use the environment variable SKIP_UI_BUILD=true to skip it, like `SKIP_UI_BUILD=true npm run test`

Reply:
Yes that part is missing in this PR. The job config needs to be updated.

Reply:
Done